enterprise/server/test/integration/remote_execution/rbetest/rbetest.go

```diff
@@ -666,7 +668,10 @@ func (r *Env) newTestAuthenticator() *testauth.TestAuthenticator {
 	users[ExecutorAPIKey] = &testauth.TestUser{
 		GroupID:       ExecutorGroup,
 		AllowedGroups: []string{ExecutorGroup},
-		Capabilities:  []akpb.ApiKey_Capability{akpb.ApiKey_REGISTER_EXECUTOR_CAPABILITY},
+		// TODO(bduffany): Don't give this API key admin rights, since API keys
+		// don't actually have admin rights in real apps.
+		GroupMemberships: []*interfaces.GroupMembership{{GroupID: ExecutorGroup, Role: role.Admin}},
```

Comment:
I'm having trouble squaring this comment (don't give admin rights) with the line below it: Role: admin. Could you elaborate on this a little bit?

Reply:
Reworded the comment to clarify. This is just a TODO to change `role.Admin` to `role.Default`, since this `TestUser` is actually representing an API key, and currently API keys cannot have `role.Admin`.

Addressing the TODO in this PR would take a bit of refactoring since the only way to get admin role currently is to authenticate over HTTP via protolet, and the RBE test env is not (yet?) set up for this.